Tools/SandboxTest.ps1

```diff
@@ -83,7 +83,19 @@ New-Item $tempFolder -ItemType Directory -ErrorAction SilentlyContinue | Out-Nul
 $WebClient = New-Object System.Net.WebClient
 
 function Get-Release {
-  $releasesAPIResponse = Invoke-RestMethod 'https://api.github.com/repos/microsoft/winget-cli/releases?per_page=100'
+  # TODO: Further improvements
+  # Sending authorized requests with GitHub CLI (aka. `gh.exe`) is much more convenient than using `Invoke-RestMethod`.
+  # It would be better integrate this script with GitHub CLI, since developers do not need creating and configuring PAT manually.
+  if ($env:GITHUB_TOKEN && $env:GITHUB_TOKEN.StartsWith('ghp_')) {
```

Comment:
IIRC only classic tokens start with prefix `ghp_`. Fine-grained tokens seem to begin with `github_pat_`. I would prefer if we don't make an assumption on prefix and make a request first to `https://api.github.com/octocat` to first check if the credentials are valid

https://docs.github.com/en/rest/authentication/authenticating-to-the-rest-api?apiVersion=2022-11-28